lib/model.js

```diff
@@ -23,7 +23,7 @@ Model.prototype.get = function(pk, store) {
 	});
```

Comment:
@giorgosera 
As a general comment can we stick to a format for our git commit messages?
ie:
- Removes
or
- Removed
?